libass/ass_priv.h

```diff
@@ -33,16 +33,18 @@ typedef enum {
 } ParserState;
 
 typedef enum {
-    SINFO_LANGUAGE     = 1 << 0,
-    SINFO_PLAYRESX     = 1 << 1,
-    SINFO_PLAYRESY     = 1 << 2,
-    SINFO_TIMER        = 1 << 3,
-    SINFO_WRAPSTYLE    = 1 << 4,
-    SINFO_SCALEDBORDER = 1 << 5,
-    SINFO_COLOURMATRIX = 1 << 6,
-    SINFO_KERNING      = 1 << 7,
+    SINFO_LANGUAGE     = 1 <<  0,
+    SINFO_PLAYRESX     = 1 <<  1,
+    SINFO_PLAYRESY     = 1 <<  2,
+    SINFO_LAYOUTRESX   = 1 <<  3,
+    SINFO_LAYOUTRESY   = 1 <<  4,
+    SINFO_TIMER        = 1 <<  5,
+    SINFO_WRAPSTYLE    = 1 <<  6,
+    SINFO_SCALEDBORDER = 1 <<  7,
+    SINFO_COLOURMATRIX = 1 <<  8,
+    SINFO_KERNING      = 1 <<  9,
     // for legacy detection
-    GENBY_FFMPEG       = 1 << 8
+    GENBY_FFMPEG       = 1 << 10
```

Comment:
Not a big deal, but personally I’d just add the new values at the end or after `SINFO_KERNING` to minimize the diff and `git blame` noise. (Maybe push `GENBY_FFMPEG` further back as well to provide some extra space.)

Reply:
Done.